docs/userguide/keywords.rst

```diff
@@ -8,19 +8,19 @@ unless you need the associated ``setuptools`` feature.
 ``include_package_data``
     If set to ``True``, this tells ``setuptools`` to automatically include any
     data files it finds inside your package directories that are specified by
-    your ``MANIFEST.in`` file.  For more information, see the section below on
```

Comment:
@alvyjudy here and in many other places (even outside of this PR) the text keeps referring to sections "below" and "above" while those are no longer in the same document. JFYI if you're up to reading through the whole documentation to find such places...